pandas/core/frame.py

```diff
@@ -2102,6 +2102,29 @@ def to_gbq(
         --------
         pandas_gbq.to_gbq : This function in the pandas-gbq library.
         read_gbq : Read a DataFrame from Google BigQuery.
+
+        Examples
+        --------
+        Example taken from `Google BigQuery documentation
+        <https://cloud.google.com/bigquery/docs/samples/bigquery-pandas-gbq-to-gbq-simple>`_
+
+        >>> import pandas_gbq   # doctest: +SKIP
```

Comment:
You can remove this import as well (and skipping setting `project_id` and `table_id` since the `df.to_gbq` is skipped

Reply:
If I skip `project_id` and `table_id` I get this:
```
flake8 error: line 10, col 11: F821 undefined name 'table_id'
flake8 error: line 10, col 32: F821 undefined name 'project_id'
```

Reply:
Yeah so you can not skip setting `project_id` and `table_id` since they are just strings